<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1292</width>
    <height>842</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Тича</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>1274</width>
       <height>824</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 218, 185);
alternate-background-color: rgb(170, 255, 127);
selection-color: rgb(255, 160, 122);
border-color: rgb(255, 201, 165);
QTabWidget::pane {
    background-color: #FFDAB9;
}</string>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Домашняя страница</string>
      </attribute>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>444</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Следующие запланированные занятия:</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>350</y>
         <width>411</width>
         <height>351</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>273</y>
         <width>624</width>
         <height>29</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Общая информация о расписании на текущую неделю:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Ученики</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QPushButton" name="Show_Student_All">
         <property name="font">
          <font>
           <family>Palatino Linotype</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;
	background-color: rgb(255, 209, 187);
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF7518;
}
</string>
         </property>
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="Del_Student">
         <property name="font">
          <font>
           <family>Palatino Linotype</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;
	background-color: rgb(255, 209, 187);
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF7518;
}
</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="Add_student">
         <property name="font">
          <font>
           <family>Palatino Linotype</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;
	background-color: rgb(255, 209, 187);
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF7518;
}
</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTableView" name="tableView">
         <property name="font">
          <font>
           <family>Palatino Linotype</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ContiguousSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_7">
      <attribute name="title">
       <string>Расписание</string>
      </attribute>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>811</width>
         <height>381</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Календарь с отображением доступных временных слотов и запланированных занятий.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Show_Student_Active">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>640</y>
         <width>441</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    border-style: outset;
    border-width: 2px;
    border-color: gray;
    padding: 4px;
	background-color: rgb(255, 209, 187);
	border-radius: 10px;


}

QPushButton:pressed {
    border-style: inset;
    background-color: #FF7518;
}
</string>
       </property>
       <property name="text">
        <string>Провести занятие</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>Финансы</string>
      </attribute>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>781</width>
         <height>301</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Таблица с записями о платежах от учеников и статусом задолженностей.</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>260</y>
         <width>761</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Кнопка &quot;Добавить платеж&quot; для фиксирования нового платежа.</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>Отчеты</string>
      </attribute>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>851</width>
         <height>331</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Выбор типа отчета (например, доходы, расписание, успеваемость учеников).
Кнопка &quot;Сформировать отчет&quot; для генерации выбранного отчета.</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_10">
      <attribute name="title">
       <string>Аналитика</string>
      </attribute>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>1141</width>
         <height>401</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Palatino Linotype</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Графики и диаграммы с ключевыми метриками, такими как количество занятий, доходы и успеваемость учеников.</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Коммуникация с учениками</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
